### ISSUE TITLE
`no-run-logic`: allow assignment

### ISSUE THREAD

**Issue by jrencz** [Gillespie59/eslint-plugin-angular#410]
The following (apparently a valid configuration according to [Angular documentation on $http caching](https://docs.angularjs.org/api/ng/service/$http#caching)) is not valid with `no-run-logic` enabled

```js
.run(function (
  $http,
  someCacheService
) {
  $http.defaults.cache = someCacheService;
})
```

It should be allowed to assign (maybe optionally?)

**Comment by tilmanpotthof** [comment]
Good point. I also think the rule `no-run-logic` is not exactly what the referenced rule by johnpapa says (https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#style-y171).

**Comment by MarcLoupias** [comment]
You shoud write a service ton configure your cache.

The AngularJS documentation is full of bad practices.
